closed/OpenJ9.gmk

```diff
@@ -225,23 +233,23 @@ endef
 # param 1 = The target directory to create or update.
 # param 2 = The source directory to copy.
 define openj9_copy_tree
-  $(call openj9_copy_tree_impl,$(strip $(abspath $1)),$(strip $(abspath $2)))
+	$(call openj9_copy_tree_impl,$(strip $(abspath $1)),$(strip $(abspath $2)))
 endef
 
 OPENJ9_MARKER_FILE := .up-to-date
 
-# Use '-m' / '--touch' to avoid file modification times.  `-m` works with older tar tools
+# Use '-m' / '--touch' to avoid file modification times (`-m` works with older tar tools).
 define openj9_copy_tree_impl
-  @$(MKDIR) -p $1
-  @$(TAR) --create --directory=$2 $(if $(wildcard $1/$(OPENJ9_MARKER_FILE)),--newer=$1/$(OPENJ9_MARKER_FILE)) --exclude-vcs . | $(TAR) --extract --directory=$1 -m
-  @$(TOUCH) $1/$(OPENJ9_MARKER_FILE)
+	@$(MKDIR) -p $1
+	@$(TAR) --create --directory=$2 $(if $(wildcard $1/$(OPENJ9_MARKER_FILE)),--newer=$1/$(OPENJ9_MARKER_FILE)) --exclude-vcs . | $(TAR) --extract --directory=$1 -m
+	@$(TOUCH) $1/$(OPENJ9_MARKER_FILE)
 endef
 
 # Temporary fix until we work out how to handle the build_jdk for cross compiles
-ifneq ($(OPENJDK_BUILD_CPU),$(OPENJDK_TARGET_CPU))
-  $(info Cross compilation detected, skipping generated_target_rules_build)
-else
+ifeq ($(OPENJDK_BUILD_CPU),$(OPENJDK_TARGET_CPU))
   $(eval $(call generated_target_rules_build,build_jdk,$(BUILD_JDK)))
```

Comment:
In openj9-openjdk-jdk8 the similar change was to use JDK_OUTPUTDIR and revert the custom-spec change that assigns JDK_OUTPUTDIR to BUILD_JDK.  That same change should apply here.

Reply:
At first I was inclined to agree, but `BUILD_JDK` is the right name to use here. In Java 9 it is defined during configuration; see common/autoconf/spec.gmk.in (which might be given via `--with-build-jdk`).